run_tests.sh

```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-py.test --cov=neuraxle/ testing/
+py.test  testing/
```

Comment:
Is this needed? I thought only the .cfg file needed to be changed for the debugging to work.